requirements.txt

```diff
@@ -1,2 +1,2 @@
-cryptography==2.7
-PyJWT>=1.6.4,<1.7
+cryptography
```

Comment:
To avoid the library from breaking because of incompatible changes in those libraries, I would leave it instead as it was. In a separate PR the libraries can be updated but better not here.

Reply:
Fair point
But since between cryptography 2.7 and 2.8 are no breaking changes I would propose `cryptography<2.9`
Same for PyJWT `PyJWT<1.8`

Reply:
Going to make that change if I'm pushing for #23